src/amcharts/Bubble.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+(function(root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "./CommonXY", "amcharts/xy", "../chart/INDChart", "css!./Bar"], factory);
+    } else {
+        root.amcharts_Bubble = factory(root.d3, root.amcharts_CommonXY, root.amcharts, root.chart_INDChart);
+    }
+}(this, function(d3, CommonXY, AmCharts, INDChart) {
+    function Bubble() {
+        CommonXY.call(this);
+        this._class = "amcharts_Bubble";
+        this._tag = "div";
+        
+        this._type = "Bubble";
+        this._gType = "column";
+    };
+    Bubble.prototype = Object.create(CommonXY.prototype);
+
+    Bubble.prototype.implements(INDChart.prototype);
+
+    Bubble.prototype.testData = function() {
```

Comment:
Its better to default to the INDChart test data (for when it changes in the future, to add more tests for things like annotations).